src/common/constants.ts

```diff
@@ -15,3 +15,5 @@ export const FEE_RECIPIENT = process.env.REACT_APP_FEE_RECIPIENT || ZERO_ADDRESS
 
 export const MAKER_FEE = process.env.REACT_APP_MAKER_FEE || '0';
 export const TAKER_FEE = process.env.REACT_APP_TAKER_FEE || '0';
+
+export const DEFAULT_FALLBACK_ICON_COLOR = '#808080';
```

Comment:
@gabitoesmiapodo Would it make sense to make this part of the `theme` variables?

Reply:
If it's going to be used elsewhere, yes it would make sense.

Reply:
Done!